collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -14,24 +14,32 @@
 def charts_template(sm):
     order = [
         'alarms',
+        'values'
     ]
 
     mappings = ', '.join(['{0}={1}'.format(k, v) for k, v in sm.items()])
     charts = {
         'alarms': {
-            'options': [None, 'Alarms ({0})'.format(mappings), 'status', 'alarms', 'alarms.status', 'line'],
+            'options': [None, 'Alarms ({0})'.format(mappings), 'status', 'Alarms', 'alarms.status', 'line'],
```

Comment:
Why did you capitalize the family?

Reply:
oh yep - that's not like how we do it for others - yep - will revert.